srml/contract/src/lib.rs

```diff
@@ -366,11 +404,11 @@ decl_event! {
 
 decl_storage! {
 	trait Store for Module<T: Trait> as Contract {
-		/// The fee required to create a contract.
+		/// The fee required to create a contract instance.  Must be at least as big as staking’s `ReclaimRebate`.
 		ContractFee get(contract_fee) config(): T::Balance = T::Balance::sa(21);
-		/// The fee charged for a call into a contract.
+		/// The base fee charged for a call into a contract.
```

Comment:
Match form as long as it's still accurate.
```suggestion
		/// The base fee charged for calling into a contract.
```